stellar/contract/round/src/internal.rs

```diff
@@ -736,6 +736,10 @@ impl IsRound for RoundContract {
         let current_ms = get_ledger_second_as_millis(env);
 
         if round.voting_start_ms <= current_ms && current_ms <= round.voting_end_ms {
+            let is_blacklisted = is_blacklisted(env, round_id, voter.clone());
+            if is_blacklisted {
+                return false;
+            }
             if round.use_whitelist_voting {
                 if round.voting_wl_list_id.is_none() {
                     return true;
```

Comment:
It appears strange to me that despite the round explicitly requires using the whitelists the function returns `true` in case the voting whilelist isn't set. IMO, it should return false in this case, since we can not ensure that the user is really whitelisted.

Reply:
I think you've fixed that behavior in another PR, https://github.com/PotLock/grantpicks/pull/251/files  , so this should be fine then.